src/main/java/seedu/address/logic/parser/ExportCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.ExportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class ExportCommandParser implements Parser<ExportCommand> {
+
+    public static final String INVALID_EXPORT_COMMAND = "Invalid command format!\n"
+            + "export: Exports the corresponding information to a text file\n"
+            + "Parameters: INFORMATION (e.g. email, phone)\n"
+            + "Example: export email";
```

Comment:
This part perhaps can declare it as `MESSAGE_USAGE` in the `ExportCommand`?
So, it follows the current exception handling approach. 
Can refer to AddCommand and AddCommandParser.

Reply:
Will do thanks